index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <title>My Portolio Project</title>
+</head>
+<body>
+
+  
+        <header class ="page-header">
+            <div>
+              <a id="logo-text" href="welcome">Welcome</a>
+            </div>  
+            <nav id="navbar">
+            <i class="fa fa-bars"></i>
```

Comment:
Great design @Shedrack-Sunday 🤝

- [ ] [Optional] Kindly make this hamburger icon a clickable link. You may surround it with an `a` tag to make it a link.